scalalib/src/mill/javalib/checkstyle/CheckstyleModule.scala

```diff
@@ -31,6 +31,9 @@ trait CheckstyleModule extends JavaModule {
       Seq("-f", checkstyleFormat()) ++
       (if (stdout) Seq.empty else Seq("-o", output.toString())) ++
       (if (leftover.value.nonEmpty) leftover.value else sources().map(_.path.toString()))
+    val jvmArgs = sys.props.get("user.language")
+      .map(lang => s"-Duser.language=$lang")
+      .toSeq
```

Comment:
Using `sys.props` here adds a hidden input to this anonymous task. It won't trigger a re-evaluating if the user changes this property. It's currently called from a command, so it should be no issue. But it's a hidden code smell.

Maybe, we should pass the language as parameter?

Reply:
Just added a `Task.Input` reading this property, `CheckstyleModule#checkstyleLanguage`